app/services/user_alerts/alert_user_about_account_verified.rb

```diff
@@ -4,11 +4,10 @@ module UserAlerts
   class AlertUserAboutAccountVerified
     def self.call(profile:)
       user = profile.user
-      sp_name = profile.initiating_service_provider&.friendly_name || APP_NAME
       user.confirmed_email_addresses.each do |email_address|
         UserMailer.with(user: user, email_address: email_address).account_verified(
           date_time: profile.verified_at,
-          sp_name: sp_name,
+          profile: profile,
```

Comment:
We start using a presenter, and it can fetch everything we need off of the profile.

In hindsight, `date_time` isn't needed here either.